templates/wordpress/files/web/index.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
Does this come from upstream, or is it vanilla ours?

Reply:
It's ours. Moving the wp web root doesn't work without a custom index.php